docs/compliance/pci/index.md

```diff
@@ -0,0 +1,366 @@
+
+|Item      |Example|
+|----------|------|
+|Username: |`adminXX@contosowebstore.com`|
+| Name: |`Global Admin Azure PCI Samples`|
+|User type:| `Subscription Administrator and Azure Active Directory Global Administrator`|
+
+* The admin account cannot read credit card information unmasked. All actions are logged.
+* The admin account cannot manage or log into SQL Database.
+* The admin account can manage Active Directory and subscription.
+
+#### Role: SQL administrator
+
+|Item      |Example|
+|----------|------|
+|Username: |`sqlAdmin@contosowebstore.com`|
+| Name: |`SQLADAdministrator PCI Samples`|
+| First name: |`SQL AD Administrator`|
+|Last name: |`PCI Samples`|
+|User type:| `Administrator`|
+
+* The sqladmin account cannot view unfiltered credit card information. All actions are logged.
+* The sqladmin account can manage SQL database.
+
+#### Role: Clerk
+
+|Item      |Example|
+|----------|------|
+|Username:| `receptionist_EdnaB@contosowebstore.com`|
+| Name: |`Edna Benson`|
+| First name:| `Edna`|
+|Last name:| `Benson`|
+| User type: |`Member`|
+
+Edna Benson is the receptionist and business manager. She is responsible for ensuring that customer information is accurate and billing is completed. Edna is the user logged in for all interactions with the Contoso Webstore demo website. Edna has the following rights: 
+
+- Edna can create and read customer information
+- Edna can modify customer information.
+- Edna can overwrite or replace credit card number, expiration, and CVV information.
+
+> In the Contoso Webstore, the user is automatically as the **Edna** user for testing the capabilities of the deployed environment.
+
+### Contoso Webstore - Estimated pricing
+
+This foundational architecture and example web application have a monthly fee structure and a usage cost per hour which must be considered when sizing the solution. These costs can be estimated using the [Azure costing calculator](https://azure.microsoft.com/en-us/pricing/calculator/). As of September 2017, the estimated monthly cost for this solution is $XXX. These costs will vary based on the usage amount and are subject to change. It is incumbent on the customer to calculate their estimated monthly costs at the time of deployment for a more accurate estimate. 
+
+This solution used the following Azure services. Details of the deployment architecture are located in the [Deployment Architecture](#deployment-architecture) section.
+
+>- Application Gateway
+>- Azure Active Directory
+>- App Service Environment
+>- OMS Log Analytics
+>- Azure Key Vault
+>- Network Security Groups
+>- Azure SQL DB
+>- Azure Load Balancer
+>- Application Insights
+>- Azure Security Center
+>- Azure Web App
+>- Azure Automation
+>- Azure Automation Runbooks
+>- Azure DNS
+>- Azure Virtual Network
+>- Azure Virtual Machine
+>- Azure Resource Group and Policies
+>- Azure Blob Storage
+>- Azure Active Directory role-based access control (RBAC)
+
+## Deployment architecture
```

Comment:
I'm still not sure what "Deployment architecture" means or how that's distinct from "Architecture" 

Can we call this section Recommendations?